oas-contract-tests/src/main/java/springfox/test/contract/oas/model/User.java

```diff
@@ -175,7 +175,7 @@ public User email(String email) {
    *
    * @return email
    */
-  @io.swagger.v3.oas.annotations.media.Schema(example = "john@email.com", description = "")
+  @io.swagger.v3.oas.annotations.media.Schema(example = "john@email.com", description = "", nullable = true)
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.